samples/create_basic_vm.py

```diff
@@ -0,0 +1,132 @@
+# Description: This sample creates basic virtual machine
```

Comment:
Please can you add Author and Copyright info here?

Reply:
added. please check